src/Strategy/Composer.hs

```diff
@@ -0,0 +1,133 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+module Strategy.Composer
+  ( discover,
+    analyze,
+    buildGraph,
+    ComposerLock (..),
+    CompDep (..),
+    Source (..),
+  )
+where
+
+import Control.Effect.Diagnostics
+import qualified Data.Map.Strict as M
+import DepTypes
+import Discovery.Walk
+import Effect.Grapher
+import Effect.ReadFS
+import Graphing (Graphing)
+import Prologue
```

Comment:
We done killed the prologue in #124.  You'll definitely need to add a few module to cover the missing ones.  Here's a few you definitely need:

- `Data.Aeson.Types`
- `Data.Foldable (find)`
- `Data.Text (Text)`